lib/diagnostic.js

```diff
@@ -51,7 +95,11 @@ db.dropDatabase();
 // House Stark.
 
 // your code begins here
-
+db.houses.deleteOne(
+  {
+    house: 'House Stark'
```

Comment:
This syntax is correct, but it will not delete anything because your collection doesn't have documents with `house` as a key (you used `name`, which is correct).